README.md

```diff
@@ -46,6 +46,15 @@ Workflow is as follows:
 - Add unit testing for desired functionality - `npm test` for testing
 - Work on functionality
 - Bump relevant version when finished `npm version [major|minor|patch]`
+- Update typescript definitions if necessary
+  * ```bash
+    # You can simply paste all this into your terminal
+    npm uninstall -g r7insight_node
+    npm pack
+    npm i -g r7insight_node-*.tgz
```

Comment:
Is this just being installed globally to get around the `Can't install package in directory with same name` error when generating TS defs or is it required to be available everywhere?